generators/client/templates/angular/src/test/javascript/spec/helpers/_activate.service.ts

```diff
@@ -0,0 +1,20 @@
+import {SpyObject} from "./spyobject";
+import {Activate} from "../../../../main/webapp/app/account/activate/activate.service";
+import Spy = jasmine.Spy;
+
+export class MockActivate extends SpyObject {
```

Comment:
instead of writing a mock for every service may be we can write a generic mockService that can be set for most of the services. it could have generic methods implemented with fakResponses